asserts/ifacedecls.go

```diff
@@ -220,7 +222,256 @@ func compileAttributeConstraints(constraints interface{}) (*AttributeConstraints
 	return &AttributeConstraints{matcher: matcher}, nil
 }
 
+type fixedAttrMatcher struct {
+	result error
+}
+
+func (matcher fixedAttrMatcher) match(context string, v interface{}) error {
+	return matcher.result
+}
+
+var (
+	AlwaysMatchAttributes = &AttributeConstraints{matcher: fixedAttrMatcher{nil}}
+	NeverMatchAttributes  = &AttributeConstraints{matcher: fixedAttrMatcher{errors.New("not allowed, never matching")}}
```

Comment:
Will that ", never matching" be surfacing to users?  People might be confused as they don't know the decision process goes through such a mechanism. Perhaps just "not allowed" would be better if that's the case.